test/integration/testMaxLobSize.js

```diff
@@ -95,6 +98,52 @@ if (process.env.RUN_MANUAL_TESTS_ONLY === 'true'){
       });
     });
 
+    describe('test put get max LOB size', function () {
+      before(async function () {
+        connection = testUtil.createConnection();
+        await testUtil.connectAsync(connection);
+        await testUtil.executeCmdAsync(connection, createTable);
+        await testUtil.executeCmdAsync(connection, allowLobExternalScan);
+      });
+
+      after(async () => {
```

Comment:
let's be consistent - in before we have `async function () ...`

Reply:
changed